src/QtHardMonForm.ui

```diff
@@ -519,6 +519,9 @@
            <property name="text">
             <string>Read</string>
            </property>
+           <property name="shortcut">
+            <string>F5</string>
```

Comment:
We also talked about <ctrl><R>. I like that because it is also a standard reload shortcut, and R can mean "read".